package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "medic-conf-test-harness",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Were you making changes from older version or our package.json updated. I believe we already have 2.1.3 released for medic-conf-test-harness. 

https://www.npmjs.com/package/medic-conf-test-harness

Reply:
Let's merge https://github.com/medic/medic-conf-test-harness/pull/111 first and make this 2.1.4?